src/main/java/org/graylog/integrations/aws/AWSClientBuilderUtil.java

```diff
@@ -88,10 +88,14 @@ public static KinesisClient buildClient(KinesisClientBuilder clientBuilder, AWSR
      * @return A fully built {@link IamClient}
      */
     public static IamClient buildClient(IamClientBuilder clientBuilder, AWSRequest request) {
+        Region iamRegion = Region.AWS_GLOBAL;
+        if (request.region().contains("gov")) {
```

Comment:
This looks good to me. This seems like the best way to make the determination for which global region to use.

@waab76 Is it possible to add a test covering this if-statement?

Reply:
Yeah, I'll do that now.